### PR TITLE
Developers guide glossary cleanup

### PR DESCRIPTION
**Pull request by derlerd-dfinity**
More consistent and technically accurate documentation.

### CODE REVIEW

File: modules/developers-guide/pages/glossary.adoc [hunk 1]
```diff
@@ -8,53 +8,45 @@ actor::
...
+  Note that the developer identity can also be used to derive an **account identifier**—similar to a Bitcoin or Ethereum address—to hold ICP tokens on the identity's behalf in the ledger canister.
 
 [[g-controller]]
 controller::
-  A controller is an identity that has special rights to manage the canister it controls.
-  For example, only a controlling identity can be used to install, upgrade, or delete the canister under its control.
-  You can specify the controller identity using the textual representation of a principal—often referred to as the principal identifier—associated with a user or with a canister.
+  A controller is a principal that has special rights to manage the canister it controls.
```
Comment (yvonneanne) [inline modules/developers-guide/pages/glossary.adoc:22]:
```suggestion
  A controller is a principal that has special rights to manage the canister smart contract it controls.
```

File: modules/developers-guide/pages/glossary.adoc [hunk 2]
```diff
@@ -8,53 +8,45 @@ actor::
...
-[[g-principal]]
-principal::
-  The first time you use the {sdk-short-name}, the `+dfx+` command-line tool creates a `+default+` developer identity for you with a public/private key pair in a PEM file. This {IC} developer identity is represented internally by a derived **principal** data type and a textual representation of the principal often referred to as your **principal identifier**.
-  The developer identity can also be used to derive an **account identifier**—similar to a Bitcoin or Ethereum address—to hold ICP tokens on your behalf in the {IC} ledger canister.
+  A physical or virtual machine that hosts all the hardware, replica software, and configuration settings required to participate in the {IC} blockchain.
 
 [[g-replica]]
 replica:: 
-  In the context of the {platform}, a replica refers to the {IC} processes (for example, the `+replica+`, `+nodemanager+`, and other lower-level {IC} protocol processes) running on a physical computer node in the network.
+  In the context of the {IC} blockchain, a replica refers to the {IC} protocol processes (for example, the `+replica+`, `+nodemanager+`) running on a node.
```
Comment (yvonneanne) [inline modules/developers-guide/pages/glossary.adoc:45]:
```suggestion
  In the context of the {IC} blockchain, a replica refers to the {IC} protocol processes (for example, the `+replica+` process or the `+nodemanager+` processes) running on a node.
```

File: modules/developers-guide/pages/glossary.adoc [hunk 3]
```diff
@@ -8,53 +8,45 @@ actor::
...
   For the {sdk-short-name}, you use the `+dfx start+` and `+dfx stop+` commands to start and stop the `+replica+` process locally to provide a local network for development.
```
Comment (yvonneanne) [inline modules/developers-guide/pages/glossary.adoc:46]: Is there now one or several processes that a make up a replica and which are started and stopped with dfx?

File: modules/developers-guide/pages/glossary.adoc [hunk 4]
```diff
@@ -8,53 +8,45 @@ actor::
...
-  On the {IC}, a wallet is a specialized application that allows you to store and retrieve your digital assets.
-  The wallet application is implemented as a canister that runs on the {IC}. A wallet enables you to manage your ICP token balances, convert ICP token into cycles, and distribute cycles to your own or other users' canisters as a means for accessing or providing internet services.
-
 [[g-wasm]]
 WebAssembly::
   https://webassembly.org/[WebAssembly] (`+Wasm+`) is a low-level computer instruction format. 
-  Because WebAssembly defines a portable,open-standard, binary format that abstracts cleanly over most modern computer hardware, it is broadly supported for programs that run on the internet. 
-  Programs written in {proglang} are compiled to WebAssembly code for execution on {IC} replicas.
+  Because WebAssembly defines a portable, open-standard, binary format that abstracts cleanly over most modern computer hardware, it is broadly supported for programs that run on the internet.
+  Programs written in {proglang} are compiled to WebAssembly code for execution on replicas.
```
Comment (yvonneanne) [inline modules/developers-guide/pages/glossary.adoc:52]:
```suggestion
  Dapps written in {proglang} are compiled to WebAssembly code for execution on the {IC} blockchain.
```
Maybe like this?

**Review by yvonneanne — APPROVED** [review]

File: modules/developers-guide/pages/glossary.adoc (hunk 3 above)
Comment (derlerd-dfinity) [reply modules/developers-guide/pages/glossary.adoc:46]: Great point this is still somewhat confusing. How about this?
```
In the context of the {IC} blockchain, a replica refers to the {IC} protocol processes running on a node.
  For the {sdk-short-name}, you can use the `+dfx start+` and `+dfx stop+` commands to start and stop a `+replica+` process that provides a local network for development.
```

Comment (derlerd-dfinity) [reply modules/developers-guide/pages/glossary.adoc:46]: Or maybe even something like this
```
In the context of the {IC} blockchain, a replica refers to the {IC} protocol processes running on a node. For the {sdk-short-name}, you can use the `+dfx start+` and `+dfx stop+` commands to start and stop a `+replica+` process that emulates a local network for development purposes.
```

File: modules/developers-guide/pages/glossary.adoc (hunk 4 above)
Comment (derlerd-dfinity) [reply modules/developers-guide/pages/glossary.adoc]: Github won't let me apply this suggestion. Will manually apply.

File: modules/developers-guide/pages/glossary.adoc (hunk 3 above)
Comment (yvonneanne) [reply modules/developers-guide/pages/glossary.adoc]: I like the second one much better, but I'd use processes in plural in the second one as well (unless it's only one?)

Comment (derlerd-dfinity) [reply modules/developers-guide/pages/glossary.adoc]: Good point. Done here 4fa6444.

**Review by yvonneanne — APPROVED** [review]

**Review by MatthiasDFN — APPROVED** [review]

**Pull request merged**